docs/source/_redirects

```diff
@@ -1 +1 @@
-# Placeholder, at the moment.
+/configuration/spaceport* /docs/router/configuration/apollo-telemetry/
```

Comment:
Curious, as a general pattern, are we meant to be adding the asterisk after any page (read: non-directory) when doing redirects like this?  I wouldn't have thought that this was necessary, but would like to understand more!

More context for me: I've understood Netlify's proxy to normalize trailing slashes off at the end of URLs and this was a page.  If there was an intent to capture other `/spaceport/*` pages (e.g., `/spaceport/cheese`) in this redirect, then I'd think `/spaceport/*` would be what we want.  While I'd  While I'd think this would match `spaceportttt`, I don't think we'd need that.

Anyhow, merging this since it solves the problem reported.  Thanks!

Reply:
Ah this stemmed entirely from uncertainty and speed! I saw that the reported broken URL was `/spaceport` (no trailing slash) and we usually see trailing slashes in docs URLs, so I worried I was going to hit one and not the other. I also figured it was a safe wildcard given our general lack of spaceport directory 😄 , but yeah probably not a best practice in general

Reply:
@StephenBarlow Got it!  All good, just wanted to check for my own understanding.  [Here's the citation](https://docs.netlify.com/routing/redirects/redirect-options/#trailing-slash) from Netlify about the normalization they do.  With that in mind, it should be wholly safe to either have or not have slash confidently and know that it doesn't matter:

> Our CDN edge nodes do URL normalization before the redirect rules kick in. This happens to make sure we can guarantee the highest possible cache hit rate and the absolute best performance for your site.
> 
> What this means for your redirect rules is that Netlify will match paths to rules regardless of whether or not they contain a trailing slash.

Reply:
Awesome thanks for checking!